modules/admin_manual/pages/release_notes.adoc

```diff
@@ -18,7 +19,15 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
-== Changes in 10.1
+== Changes in 10.1.1
+
+ownCloud Server 10.1.1 is a hotfix follow-up release that takes care of https://github.com/owncloud/core/issues/34851[an issue with loading updated apps]. Instead of updating the app versions to their new values in the database, the old version value is written causing the process to repeat with every request.
+
+This issue can cause high load on the database, especially in large installations. If you have already upgraded to 10.1.0, please consider upgrading to 10.1.1. You can expect minimal downtime for the upgrade to this patch release.
```

Comment:
I suggest using stronger wording than "please consider upgrading", such as "we strongly recommend upgrading".

Reply:
Thx, changed.